pandapower/converter/cim/cim2pp/converter_classes/generators/synchronousMachinesCim16.py

```diff
@@ -145,3 +148,39 @@ def _prepare_synchronous_machines_cim16(self) -> pd.DataFrame:
             'minOperatingP': 'min_p_mw', 'maxOperatingP': 'max_p_mw', 'minQ': 'min_q_mvar', 'maxQ': 'max_q_mvar',
             'ratedPowerFactor': 'cos_phi', 'referencePriority': 'slack_weight'})
         return synchronous_machines
+
+    def _create_gen_characteristics_table(self, syn_gen_df_origin) -> pd.DataFrame:
+        if 'id_characteristic_table' not in syn_gen_df_origin.columns:
```

Comment:
If this column is referring to the gen_capability_curve_table can we maybe rename it to e.g. "id_capability_curve_table"?